code/controllers/subsystem/persistence.dm

```diff
@@ -337,33 +400,33 @@ SUBSYSTEM_DEF(persistence)
 
 	WRITE_FILE(frame_path, frame_json)
 
-/datum/controller/subsystem/persistence/proc/CollectTrophies()
-	var/json_file = file("data/npc_saves/TrophyItems.json")
+///Collects trophies from all existing trophy cases.
+/datum/controller/subsystem/persistence/proc/collect_trophies()
+	for(var/trophy_case in GLOB.trophy_cases)
+		save_trophy(trophy_case)
+
+	var/json_file = file("data/trophy_items.json")
 	var/list/file_data = list()
-	file_data["data"] = remove_duplicate_trophies(saved_trophies)
+	var/list/converted_data = list()
+
+	for(var/datum/trophy_data/data in saved_trophies)
+		converted_data += list(data.to_json())
```

Comment:
Where did the de duplication go ?

Reply:
I think months ago i removed it thinking "I will rework it later" and then i just, forgot. Will remake it.

Reply:
Tested, readded.